### PR TITLE
Classic Menu Bar implemented

### PR DESCRIPTION
**Pull request by m-e-l-u-h-a-n**
implemented the idea of classic menu bar only generalisation for n items left

### CODE REVIEW

File: classic menu bar/css/style.css [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  bottom: 7px;
+  color: #ffffff;
+  padding-top: 4px;
+}
+
+.icon-container {
+  position: absolute;
+  margin-right: 0px;
+  padding-right: 0px;
+  bottom: 6px;
```
Comment (king-11) [inline classic menu bar/css/style.css:65]: add pointer-events: auto;
and cursor: pointer;

File: classic menu bar/css/style.css [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+}
+
+.item.open:nth-child(1) {
+  position: absolute;
+  bottom: calc(55px*1);
+  transform: translateX(calc(-6*0*0px - 200px));
+  transition: transform 0.7s;
+}
```
Comment (king-11) [inline classic menu bar/css/style.css:75]: great work although alternatively, js could have lessened writing out everything.

File: classic menu bar/css/style.css [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+  bottom: calc(55px*7);
+  transform: translateX(calc(-6*6*6px - 200px));
+  transition: transform 0.7s;
+}
+
+#item-1 {
+  position: absolute;
+  right: -200px;
+  bottom: calc(55px*1);
+}
```
Comment (king-11) [inline classic menu bar/css/style.css:123]: same for these could have gone with js too.

**Review by king-11 — CHANGES_REQUESTED** [review]
just add the pointer-events rest looks great.

File: classic menu bar/css/style.css (hunk 2 above)
Comment (m-e-l-u-h-a-n) [reply classic menu bar/css/style.css:75]: Yeah, you are right i did not use it because for this work i thought of utilizing powers of sass :smile:

**Pull request merged**